data/src/main/java/pl/edu/agh/papaya/model/LocalDateTimePeriod.java

```diff
@@ -26,20 +26,29 @@ public LocalDateTimePeriod() {
         this.end = this.start;
     }
 
+    public LocalDateTime getStart() {
+        return start;
+    }
+
     public void setStart(LocalDateTime start) {
         if (start.isAfter(this.end)) {
             throw new DateTimeException("Start date cannot be after the end date.");
         }
         this.start = start;
     }
 
+    public LocalDateTime getEnd() {
+        return end;
+    }
+
     public void setEnd(LocalDateTime end) {
         if (end.isBefore(this.start)) {
-            throw new DateTimeException("Start date cannot be after the end date.");
+            throw new DateTimeException("End date cannot be before the start date.");
         }
         this.end = end;
     }
 
+    @SuppressWarnings("checkstyle:HiddenField")
     public void set(LocalDateTime start, LocalDateTime end) {
         if (end.isBefore(start)) {
             throw new DateTimeException("Start date cannot be after the end date.");
```

Comment:
A tu bym zmieniła na `throw new DateTimeException("Start date must be before the end date.");` i już nic nie będzie na nas ostrzegał